Source/WebCore/PAL/pal/spi/cocoa/WritingToolsSPI.h

```diff
@@ -25,6 +25,17 @@
 
 #pragma once
 
+#if ENABLE(WRITING_TOOLS)
+
 #if USE(APPLE_INTERNAL_SDK)
-#import <WebKitAdditions/UnifiedTextReplacementSoftLinkAdditions.h>
+
+#import <WritingTools/WTSession_Private.h>
+#import <WritingTools/WritingTools.h>
+
+#else
+
+#error Symbols must be forward declared once used with non-internal SDKS.
```

Comment:
This is kind of a weird admission to check in to the open-source project—let's make sure the work to declare these SPI symbols is tracked.